frontend/packages/console-shared/src/components/status/PopoverStatus.tsx

```diff
@@ -22,9 +22,7 @@ const PopoverStatus: React.FC<PopoverStatusProps> = ({
       isVisible={isVisible}
       shouldClose={shouldClose}
     >
-      <Button variant="link" isInline>
-        {statusBody}
-      </Button>
+      <Button variant="plain">{statusBody}</Button>
```

Comment:
This component is used in about 5 places. Do we know if this could break the layout in any of those places?

Reply:
I didn't see any issues during my testing, which included a lot of the places that `GeneralStatus` is used. I decided to just fix it in the kubevirt plugin anyway, since the problem is only taking place there.